app/routes.py

```diff
@@ -1,2 +1,52 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+
+class Planet:
+    def __init__(self, id, name, description, distance):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.distance = distance
+
+    def to_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            description=self.description,
+            distance=self.distance,
+        )
+
+
+planet_list = [
+    Planet(1, "Mercury", "tiny and hot", 0.387),
+    Planet(2, "Venus", "big and hot", 0.722),
+    Planet(3, "Earth", "home", 1.0),
+]
+
+bp = Blueprint("planets", __name__, url_prefix="/planet")
+
+
+@bp.route("", methods=["GET"])
+def handle_planets():
+    planets_list = []
+    for planet in planet_list:
+        planets_list.append(planet.to_dict())
```

Comment:
Nice use of the helper instance method! 

I know I demo'd a for loop, but you can also use list comp here like:

planets_response = [planet.to_json() for planet in planets]